### PR TITLE
Fixing go get doc

### PR DESCRIPTION
**Pull request by frezbo**

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
Also could you sign your commit?

**Comment by frezbo** [comment]
@AkihiroSuda  signed and  updated to include `go-task` installation

File: README.md [hunk 1]
```diff
@@ -52,7 +52,8 @@ $ cd usernetes
 ## Install from source
 
 ```console
-$ go get https://github.com/go-task/task
+$ go get github.com/go-task/task
+$ (cd $GOPATH/src/github.com/go-task/task && ./install-task.sh)
```
Comment (AkihiroSuda) [inline README.md]: go get github.com/go-task/task/cmd/task

Comment (frezbo) [reply README.md]: ahh silly me :D

**Pull request merged**